CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ We use semantic versioning in some slight variation until our feature set has st
 * The `PATCH_LEVEL` is used for regular CD releases which add new features and bug fixes.
 
 After this we will switch probably to real [Semantic Versioning 2.0.0](http://semver.org/)
+###3.5.40
+* Feature 1300: Added support to create pullSecret in buildConfig when pulling from private registry in Openshift.
```

Comment:
I don't see any issue there.

Reply:
@piyush1594 Is it for me? I didn't get what you mean? You mean issue opened for this change? Yes, i had opened #1293

Reply:
You need to mention Feature #1293:

Reply:
Oh ok. Will push the changes.

Reply:
Done. Changes pushed.